H4LCppModule.py

```diff
@@ -59,31 +63,54 @@ def beginJob(self):
         pass
 
     def endJob(self):
-        print("PassTrig: "+str(self.passtrigEvts)+" Events")
-        print("Pass4eCut: "+str(self.worker.cut4e)+" Events")
+        print("\n========== Print Cut flow table  ====================\n")
+        print("{:27}:{:7} {}".format("Total: ", str(self.noCutsEvts), " Events"))
+        print("{:27}:{:7} {}".format("PassTrig: ", str(self.passtrigEvts), " Events"))
+        print("{:27}:{:7} {}".format("Pass4eCut: ", str(self.worker.cut4e), " Events"))
         print("Pass4eGhostRemoval: "+str(self.worker.cutghost4e)+" Events")
         print("Pass4eLepPtCut: "+str(self.worker.cutLepPt4e)+" Events")
         print("Pass4eQCDSupress: "+str(self.worker.cutQCD4e)+" Events")
-        print("PassmZ1mZ2Cut_4e: "+str(self.worker.cutZZ4e)+" Events")
-        print("Passm4l_105_160_Cut_4e: "+str(self.worker.cutm4l4e)+" Events")
-        print("Pass4muCut: "+str(self.worker.cut4mu)+" Events")
+        print("{:27}:{:7} {}".format("PassmZ1mZ2Cut_4e: ", str(self.worker.cutZZ4e), " Events"))
+        print("{:27}:{:7} {}".format("Passm4l_105_160_Cut_4e: ", str(self.worker.cutm4l4e), " Events"))
+        print("{:27}:{:7} {}".format("Pass4muCut: ", str(self.worker.cut4mu), " Events"))
         print("Pass4muGhostRemoval: "+str(self.worker.cutghost4mu)+" Events")
         print("Pass4muLepPtCut: "+str(self.worker.cutLepPt4mu)+" Events")
         print("Pass4muQCDSupress: "+str(self.worker.cutQCD4mu)+" Events")
```

Comment:
Update the above three lines similar pattern what we used for all others. I mean like below

```python
print("{:27}:{:7} {}".format("Pass4eCut: ", str(self.worker.cut4e), " Events"))
```